### PR TITLE
pprof server support

### PR DESCRIPTION
**Pull request by morrisson**
For debugging long-term stability of gohan, pprof is useful profiling
tool. This patch enables gohan also starts server for pprof with
configuration in config file.

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -185,6 +186,14 @@ func NewServer(configFile string) (*Server, error) {
 	}
 	log.Info("logging initialized")
 
+	pprof := config.GetString("pprof", "")
+	if pprof != "" {
+		go func() {
+			log.Info("pprof server started at " + pprof)
```
Comment (vozhyk-) [inline server/server.go]: `Starting pprof server at `? It hasn't been started yet.

File: docs/configuration.md [hunk 2]
```diff
@@ -227,6 +230,17 @@ Note: DO NOT USE * configuraion in production deployment.
   cors: "*"
 ```
 
+## Profiling
+
+Gohan can run pprof server for profiling. To start pprof server, you need to
```
Comment (vozhyk-) [inline docs/configuration.md]: `a pprof server`, `To start the pprof server`

File: docs/configuration.md [hunk 3]
```diff
@@ -227,6 +230,17 @@ Note: DO NOT USE * configuraion in production deployment.
...
+specify pprof address with host:port. If you do not specify an address or specify
+empty string, pprof server does not start.
```
Comment (vozhyk-) [inline docs/configuration.md]: `an empty string`

**Comment by nati** [comment]
close this PR due to inactivity. if you still need this, plz reopen

**Pull request closed without merging**